src/Templates/src/templates/maui-contentview-csharp/NewContent1.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Maui.Controls;
+
+namespace MauiApp1
+{
+	public class NewContent1 : ContentView
+    {
+        public NewContent1()
```

Comment:
This line and the one after it have an extra indentation and should be fixed.

```suggestion
  {
      public NewContent1()
```

Reply:
Actually it looks like the indentation here is somewhat inconsistent and mixes spaces and tabs? I think that should be normalized one way or another?

Reply:
fixed